eng/build.yml

```diff
@@ -33,13 +33,13 @@ jobs:
     pool:
       # Public Linux Build Pool
       ${{ if and(eq(parameters.osGroup, 'Linux'), eq(variables['System.TeamProject'], 'public')) }}:
-        name:  NetCorePublic-Pool
-        queue: BuildPool.Ubuntu.1804.Amd64.Open
+        name:  NetCore1ESPool-Svc-Public
+        demands: ImageOverride -equals Build.Ubuntu.1804.Amd64.Open
```

Comment:
Do you know what the space guarantee is on these pools? Assets for this build get quite large. I've seen it fail in runtime and this repo can collect a lot of test assets that are expensive.

Reply:
It looks like the temp disks on these machines should be 100GB. Does that sound like it's enough space?

Reply:
It is enough if it is aggressively cleaned. Runtime had some issues with it. Let's just merge and pressure 1ES if this is not the case, since this is not something that we have a lot of leeway in.